### PR TITLE
add comments for CIDR ranges

### PR DESCRIPTION
**Pull request by freephile**
just wanted to test everything out to prep for something more useful

This commit adds only comments.

### CODE REVIEW

**Comment by jeffw16** [comment]
Wait, we can remove the `"` and PHP recognizes the CIDR ranges?

**Review by jeffw16 — CHANGES_REQUESTED** [review]
Please address my question about removing the quotes from the CIDR range.

**Comment by freephile** [comment]
It was a typo.

Sorry!

I wonder about adding linting on commit to flag and reject bad commits early. This commit was made in the browser so not using the visual linting help of VSCode

**Comment by freephile** [comment]
I'll amend the commit when I am back at my office

File: _sources/canasta/CanastaDefaultSettings.php
Comment (freephile) [inline _sources/canasta/CanastaDefaultSettings.php:1]: The purpose of this commit is to 

1. order the CIDR blocks numerically
2. add comments that remind you what the block notation is equal to
3. make my first commit to learn the project tooling and workflow without complication :-)

This branch could be deleted, and I would take no offense.

**Review by jeffw16 — APPROVED** [review]

**Pull request merged**